[2024-09-03T08:21:10.672+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: k9_etl_dag_v2.check_updates manual__2024-09-03T08:21:06.322959+00:00 [queued]>
[2024-09-03T08:21:10.679+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: k9_etl_dag_v2.check_updates manual__2024-09-03T08:21:06.322959+00:00 [queued]>
[2024-09-03T08:21:10.679+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-03T08:21:10.688+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): check_updates> on 2024-09-03 08:21:06.322959+00:00
[2024-09-03T08:21:10.693+0000] {standard_task_runner.py:57} INFO - Started process 322 to run task
[2024-09-03T08:21:10.695+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'k9_etl_dag_v2', 'check_updates', 'manual__2024-09-03T08:21:06.322959+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/k9_etl_dag_v2.py', '--cfg-path', '/tmp/tmpvjy7wa7l']
[2024-09-03T08:21:10.696+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask check_updates
[2024-09-03T08:21:10.786+0000] {task_command.py:415} INFO - Running <TaskInstance: k9_etl_dag_v2.check_updates manual__2024-09-03T08:21:06.322959+00:00 [running]> on host 52afa4428dd5
[2024-09-03T08:21:11.027+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='k9_etl_dag_v2' AIRFLOW_CTX_TASK_ID='check_updates' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T08:21:06.322959+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T08:21:06.322959+00:00'
[2024-09-03T08:21:11.052+0000] {python.py:194} INFO - Done. Returned value was: No new facts found.
[2024-09-03T08:21:11.068+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=k9_etl_dag_v2, task_id=check_updates, execution_date=20240903T082106, start_date=20240903T082110, end_date=20240903T082111
[2024-09-03T08:21:11.088+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-03T08:21:11.119+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
